input/pages/task-launch.md

```diff
@@ -0,0 +1,103 @@
+### Task profile for launch SMART applications (EXPERIMENTAL)
+
+
+This section defines a set of profiles of the FHIR Task resource that requests launch of a SMART application. These tasks could be used to instruct staff to run an application or as a result of a Clinical Reasoning deployment.
+
+Two profiles are defined:
+
+* [task-ehr-launch](StructureDefinition-task-ehr-launch.html), requests an EHR launch with optional appContext.
+* [task-standalone-launch](StructureDefinition-task-standalone-launch.html), requests a standalone launch.
+
+#### Requesting an EHR launch
+
+A Task with the profile [task-ehr-launch](StructureDefinition-task-ehr-launch.html), requests an EHR launch with optional appContext.
+
+The subject field, if present indicates the patient resource to be used in the launch context.

```

Comment:
```suggestion
The `Task.for` field, if present indicates the Patient resource to be used in the launch context.

```

I think this correction is consistent with the intent + example below?

Reply:
Correct